tesseract_core/runtime/cli.py

```diff
@@ -347,6 +349,16 @@ def _create_user_defined_cli_command(
 
     options.extend(
         [
+            click.Option(
+                ["--input-path"],
+                type=click.STRING,
+                help=(
+                    "Input path from which to read files. "
+                    "To be used in conjunction with InputFileReferences in the Tesseract InputSchema. "
+                    "See examples/filereference for a usage example."
```

Comment:
What examples/filereference is will not be clear to someone who pip installed tesseract-core. I'd just remove this.
```suggestion
```